src/pages/home/HomePage.js

```diff
@@ -67,20 +68,25 @@ const HomePage = ({cnt=3}) => {
   ];
 
   const [currentProfileIndex, setCurrentProfileIndex] = useState(0);
+  const [showMoreProfiles, setShowMoreProfiles] = useState(false);
 
   const handleNextProfile = () => {
     if (currentProfileIndex < profileDataList.length - 1) {
       setCurrentProfileIndex(currentProfileIndex + 1);
+    } else {
+      setShowMoreProfiles(true);
     }
   };
 
   const handlePrevProfile = () => {
+    setShowMoreProfiles(false);
     if (currentProfileIndex > 0) {
       setCurrentProfileIndex(currentProfileIndex - 1);
     }
   };
```

Comment:
```suggestion

  const handlePrevProfile = () => {
    if (showMoreProfiles) {
      setShowMoreProfiles(false);
    } else if (currentProfileIndex > 0) {
      setCurrentProfileIndex(currentProfileIndex - 1);
    }
  };
```

Show More에서 다시 datalist로 돌아올때 (-)를 한번 더해서 2개 이전의 카드가 보이네요!
이렇게 수정하시면 잘동작하는 것 같네요!